src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -3,9 +3,27 @@ package sbttravisci
 import sbt._, Keys._
 
 object TravisCiPlugin extends AutoPlugin {
+
+  object autoImport {
+    val isTravisBuild = settingKey[Boolean]("Flag indicating whether the current build is running under Travis")
+  }
+
+  import autoImport._
+
   override def requires = plugins.JvmPlugin
   override def trigger  = allRequirements
+
+  override def globalSettings = Seq(
+    isTravisBuild := sys.env.get("TRAVIS").isDefined)
+
   override def buildSettings = Seq(
+    scalaVersion := {
+      if (isTravisBuild.value)
+        sys.env.get("TRAVIS_SCALA_VERSION").get
+      else
+        crossScalaVersions.value.last   // sort .travis.yml versions in ascending order
```

Comment:
Turns out that:

> The jobs which are explicitly included inherit the first value of the expansion keys defined.

https://docs.travis-ci.com/user/build-matrix/#explicitly-included-jobs-inherit-the-first-value-in-the-array

This includes (IIUC) jobs added with `matrix.include`, `job.include` or build stages.

So... in hindsight, the default should be `.head` not `.last`...  Not sure what to do.

Reply:
Ahhhh, interesting! Good point. I think it's worth changing, but obviously it will require some sort of version bump. Probably 2.0?

Reply:
Yeah.  I don't have an immediate desire, but we can go that way if need be.